common/src/web/formPage.html

```diff
@@ -190,5 +190,8 @@
 </form>
 
 <input id="vsearchGadget" name="SearchableText" type="text" size="18" value="" title="Hvad søger du?" accesskey="4" class="inputLabel" />
+
+<button id="pointerEventsNone" style="pointer-events: none;"><span>I'm an effectively disabled button</span></button>
```

Comment:
This is wrong. This is not a disabled button.
It does only not receive any pointer events. It can still be triggered by keyboard.

Also see https://github.com/SeleniumHQ/selenium/issues/14427#issuecomment-2307123929

Reply:
https://github.com/SeleniumHQ/selenium/issues/14427#issuecomment-2307256232